### PR TITLE
Spell check

### PR DESCRIPTION
**Pull request by Sushant012**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Contributing to Kubernetes Documentation
 
-**First off, thanks for taking the time to contribute!**
+**First of all, thanks for taking the time to contribute!**
```
Comment (irvifa) [inline CONTRIBUTING.md:3]: I think “First off” is already valid in en language.

**Review by irvifa — COMMENTED** [review]
Please sign your CLA first

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
 
 The following is a set of guidelines for contributing to Kubernetes documentation, hosted at [Kubernetes.io](http://kubernetes.io/).
-These are just guidelines, not rules. Use your best judgment, and feel free to propose changes to this document in a pull request.
+These are just guidelines, not rules. Use your best judgement, and feel free to propose changes to this document in a pull request.
```
Comment (irvifa) [inline CONTRIBUTING.md:6]: Judgment is also valid spelling form, there’s two possibilities of this word whether it’s judgment or judgement.

**Comment by jimangel** [comment]
We are not participating in Hactoberfest.

/close

**Pull request closed without merging**